src/views/inline_equation.ts

```diff
@@ -32,14 +34,19 @@ export class InlineEquationView<PropsType extends BaseNodeProps>
 
   public updateContents = () => {
     if (!isRejectedInsert(this.node)) {
-      const { SVGRepresentation } = this.node.attrs
+      const { contents, format } = this.node.attrs
 
       while (this.dom.hasChildNodes()) {
         this.dom.removeChild(this.dom.firstChild as ChildNode)
       }
 
-      if (SVGRepresentation) {
-        const fragment = sanitize(SVGRepresentation, {
+      const svgContent =
```

Comment:
I think we should restucture this a bit. We should have a `renderMath`, which accepts the contents and the format and it internally decides what to do with it.

It would also be great if we import mathjax dynamically. What I'm thinking of is something like this:
```
export const renderMath = (math, format) => {
    if (format === 'tex') {
        const { convertTexToSVG } = import('./lib/tex-to-svg')
        return convertTexToSVG(math)
    }
    ...
}
```

I think we would need some promise logic for this to work, but it should be doable.